tests/test_quality_metrics.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+from image_similarity_measures import quality_metrics
+
+
+def test_rmse(test_array1, test_array2):
+    rmse = quality_metrics.rmse(test_array1, test_array2)
+    assert float(rmse) == 0.0636659786105156
```

Comment:
since we might encounter precision issues (lots of number after decimal point), I would recommend using sth like [assert_almost_equal](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_almost_equal.html)